### PR TITLE
Globals setup in php.ini

### PR DESCRIPTION
**Pull request by ogarbe**
Now if you have global defined in config.json, all bool value can be setup using php.ini

tested with phalcon 2.0 with orm.xxx parameters

can also help for https://github.com/phalcon/zephir/issues/982

can also be used for https://github.com/phalcon/zephir/issues/660 (i don't see the use of separating global and ini-entries)

### CODE REVIEW

File: Library/Code/Builder/Struct.php [hunk 1]
```diff
@@ -139,11 +139,12 @@ public function getCDefault($name, $global)
 
             case 'boolean':
             case 'bool':
+return '';
```
Comment (ogarbe) [inline Library/Code/Builder/Struct.php]: pretty bad but didn't manage not to overwrite value otherwise

**Comment by ogarbe** [comment]
@andresgutierrez what do you think of this ?

**Comment by andresgutierrez** [comment]
Could you please rebase? What would happen if there is a default value in config.json?

**Comment by ogarbe** [comment]
the default value is used from config.json, and can be override in php.ini

**Comment by andresgutierrez** [comment]
Thanks

**Comment by leo108** [comment]
Is it possible to support string value?

**Pull request merged**